sdk/example/src/index.ts

```diff
@@ -0,0 +1,17 @@
+import * as testkube from "@testkube/sdk";
+
+testkube.initTestkube({
+  url: "https://demo.testkube.io/results/v1",
+});
+
+const main = async () => {
```

Comment:
Probably it's better to add information in the `README.md` file how to use it, instead of the example package.